packages/gnome_terminal.rb

```diff
@@ -3,43 +3,29 @@
 class Gnome_terminal < Package
   description 'The GNOME Terminal Emulator'
   homepage 'https://wiki.gnome.org/Apps/Terminal'
-  @_ver = '3.38.2'
-  version @_ver
+  version '3.39.90'
   compatibility 'all'
-  source_url "https://github.com/GNOME/gnome-terminal/archive/#{@_ver}.tar.gz"
-  source_sha256 'cb2a3ca5fde192998f875b2d1b4cc1caca899d936714fe9324cb66735abf68a0'
-
-  binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gnome_terminal-3.38.2-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gnome_terminal-3.38.2-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/gnome_terminal-3.38.2-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gnome_terminal-3.38.2-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-     aarch64: '274701d4a532d9b2db1ba69eb6c2d6ba1942e7c950029af5c1cfe52cc20d951d',
-      armv7l: '274701d4a532d9b2db1ba69eb6c2d6ba1942e7c950029af5c1cfe52cc20d951d',
-        i686: '3234511570b9fbe2a8a77367ef92619275c35753493fcac240635e09e1bd91db',
-      x86_64: '00cb44ba43b4f1059dbb8d769d25f8a86b627217d44ec139fde44a2bdc48ed10',
-  })
+  source_url "https://download.gnome.org/sources/gnome-terminal/3.39/gnome-terminal-3.39.90.tar.xz"
```

Comment:
Why the source url change?

Reply:
To upgrade? Lol

Reply:
I mean you changed the source from:
```
source_url "https://github.com/GNOME/gnome-terminal/archive/#{@_ver}.tar.gz"
```
to:
```
source_url "https://download.gnome.org/sources/gnome-terminal/3.39/gnome-terminal-3.39.90.tar.xz"
```
It's coming from gnome.org instead of github.com.

Reply:
Because a tarball from gnome.org seemed more ready for release than a github raw source snapshot